Adyen/Constants/ClientConfig.cs

```diff
@@ -41,7 +41,7 @@ public class ClientConfig
         public static string MarketPayFundApiVersion = "v5";
         public static string MarketPayAccountApiVersion = "v5";
         public static string HopApiVersion = "v1";
-        public static string RecurringApiVersion = "v25";
+        public static string RecurringApiVersion = "v49";
```

Comment:
wow thats a huge bumb, is there only this many change in model? I would expect more model changes from such a bump

Reply:
Right, I updated the models based on the open api specs for v49